go/vt/sqlparser/sql.y

```diff
@@ -229,6 +230,22 @@ insert_statement:
     $$ = &Insert{Comments: Comments($2), Ignore: $3, Table: $4, Columns: cols, Rows: Values{vals}, OnDup: OnDup($7)}
   }
 
+replace_statement:
+  REPLACE comment_opt into_table_name column_list_opt row_list
```

Comment:
You can also change `INSERT` into `insert_or_replace` and make it return the correct string.